os/include/unistd.h

```diff
@@ -51,6 +51,7 @@
  ****************************************************************************/
 /**
  * @defgroup UNISTD_KERNEL UNISTD
+ * @brief Provides APIs for Standard symbolic constants and Types
```

Comment:
1. Standard Symbolic Constants and Types
2. Standard Symbolic constants and Types
3. Standard Symbolic Constants And Types
Which one should I use?